source/lib/vagrant-openstack-provider/version_checker.rb

```diff
@@ -16,21 +16,39 @@ class VersionChecker
       #
       attr_accessor :status
 
+      #
+      # boolean attribute to disbale version checker
+      #
+      attr_accessor :check_enabled
+
       def initialize
         @status = nil
+        @check_enabled = true
+
+        check = ENV['VAGRANT_OPENSTACK_VERSION_CKECK']
+        @check_enabled = false if check && check.upcase == 'DISABLED'
```

Comment:
Could be simplified to just a single `@check_enabled = !check || check.upcase != 'DISABLED'`